src/com/projectkorra/projectkorra/ability/FireAbility.java

```diff
@@ -203,6 +203,38 @@ public static void playLightningbendingSound(final Location loc) {
 			}
 		}
 	}
+
+	public static void playLightningbendingChargingSound(final Location loc) {
+		if (getConfig().getBoolean("Properties.Fire.PlaySound")) {
+			final float volume = (float) getConfig().getDouble("Properties.Fire.LightningCharge.Volume");
+			final float pitch = (float) getConfig().getDouble("Properties.Fire.LightningCharge.Pitch");
+
+			Sound sound = Sound.BLOCK_BEEHIVE_WORK;
+			try {
+				sound = Sound.valueOf(getConfig().getString("Properties.Fire.LightningCharge.Sound"));
+			} catch (final IllegalArgumentException exception) {
+				ProjectKorra.log.warning("Your current value for 'Properties.Fire.LightningCharge.Sound' is not valid.");
+			} finally{
+				loc.getWorld().playSound(loc, sound, volume, pitch);
+			}
+		}
+	}
+	
+	public static void playLightningbendingHitSound(final Location loc) {
+		if (getConfig().getBoolean("Properties.Fire.PlaySound")) {
+			final float volume = (float) getConfig().getDouble("Properties.Fire.LightningHit.Volume");
+			final float pitch = (float) getConfig().getDouble("Properties.Fire.LightningHit.Pitch");
+
+			Sound sound = Sound.ENTITY_LIGHTNING_BOLT_THUNDER;
+			try {
+				sound = Sound.valueOf(getConfig().getString("Properties.Fire.LightningHit.Sound"));
+			} catch (final IllegalArgumentException exception) {
+				ProjectKorra.log.warning("Your current value for 'Properties.Fire.LightningHit.Sound' is not valid.");
+			}	finally {
```

Comment:
There should be a space between the closing brace of the catch and finally, not a tab.

```java
} finally {
```